embulk-standards/src/main/java/org/embulk/standards/JsonParserPlugin.java

```diff
@@ -128,6 +152,78 @@ public void run(TaskSource taskSource, Schema schema, FileInput input, PageOutpu
         }
     }
 
+    private static boolean isUseCustomSchema(PluginTask task) {
```

Comment:
nitpick: sounding a bit unnatural. How about `isSchemaCustomized` or `isUsingCustomSchema`?

Reply:
Chose the second one (ref: 57938d5)